packages/flutter/lib/src/material/pickers/input_date_picker.dart

```diff
@@ -227,54 +223,60 @@ class _InputDatePickerFormFieldState extends State<InputDatePickerFormField> {
     return OrientationBuilder(builder: (BuildContext context, Orientation orientation) {
       assert(orientation != null);
 
-      return Container(
-        padding: const EdgeInsets.symmetric(horizontal: 24),
-        height: orientation == Orientation.portrait ? _inputPortraitHeight : _inputLandscapeHeight,
-        child: Column(
-          children: <Widget>[
-            const Spacer(),
-            TextFormField(
-              decoration: InputDecoration(
-                border: const UnderlineInputBorder(),
-                filled: true,
-                // TODO(darrenaustin): localize 'mm/dd/yyyy' and 'Enter Date'
-                hintText: widget.fieldHintText ?? 'mm/dd/yyyy',
-                labelText: widget.fieldLabelText ?? 'Enter Date',
-              ),
-              validator: _validateDate,
-              inputFormatters: <TextInputFormatter>[
-                // TODO(darrenaustin): localize date separator '/'
-                _DateTextInputFormatter('/'),
-              ],
-              keyboardType: TextInputType.datetime,
-              onSaved: _handleSaved,
-              onFieldSubmitted: _handleSubmitted,
-              autofocus: widget.autofocus,
-              controller: _controller,
-            ),
-            const Spacer(),
-          ],
+      return TextFormField(
+        decoration: InputDecoration(
+          border: const UnderlineInputBorder(),
+          filled: true,
```

Comment:
@darrenaustin Why is this hardcoded? If I have InputDecorationTheme with filled: false, I expect it to be false, but now it's not the case.

Reply:
It is following the Material Design for this picker, which calls for it to be filled. That said, we should probably defer to the ambient `InputDecorationTheme` if one is set for this. We have a future theming pass that we will be doing on the date pickers that will allow for more customization, but I might be able to put together a small PR to fix this one issue.

Reply:
@darrenaustin Thanks. It would be great for my use case.